airtable/plugin.py

```diff
@@ -159,19 +168,26 @@ def airtable_record_list(
     base_id: AirtableBaseID,
     table_id: AirtableTableID,
     include_fields: Optional[set[str]] = None,
+    view_id: Optional[AirtableViewID] = None,
 ) -> list[AirtableRecord]:
     """
-    Return results of an Airtable `list records` API call with the option of choosing
-    which fields to include in the returned records.  The include_fields fields are
-    required, so populate it to ensure that the returned records have the fields you
-    will access by name.
-
-    If `include_fields` is not `None`, only the fields specified in `include_fields`
-    will be included.  If `include_fields` is `None`, all fields are included.
+    Returns the results of an Airtable `list records` API call, allowing selection of
+    specific fields to include in the returned records. Fields present in the schema
+    but not assigned a value will not be returned.
+
+    Important:
+    - If `include_fields` is specified (not `None`), only the fields listed in `include_fields`
+    will be included in the results.
+    - If `include_fields` is `None`, all fields that have assigned values will be included,
+    excluding any fields that are defined in the schema but are empty.
```

Comment:
"excluding" is confusing here, as the restrictive clause makes it have no intersection with "all fields that have assigned values" anyway.  That is, "excluding" isn't describing anything that's excluded from the defined set.